src/components/tables/RowActions/AccessGrants/RevokeGrant.tsx

```diff
@@ -78,8 +81,11 @@ function RevokeGrant({
     useEffect(() => {
         if (progress === ProgressStates.SUCCESS) {
             incrementSuccessfulTransformations();
+            if (mutate) {
+                void mutate();
+            }
```

Comment:
This can end up making multiple needless calls but the quickest way to handle this right now.